_rules/6cbcbe.md

```diff
@@ -0,0 +1,269 @@
+---
+name: Links with identical accessible names serve equivalent purpose
+test_type: atomic
+
+description: |
+  This rule checks that links with identical accessible names serve an equivalent purpose.
+
+success_criterion: 
+- 2.4.9 # Link Purpose (Link Only)
+
+test_aspects:
+- DOM Tree
+- CSS Styling
+
+authors:
+- Anne Thyme Nørregaard
+---
+
+## Test Procedure
+
+### Applicability
+
+This rule applies to any two or more HTML or SVG elements that have the [semantic role](#semantic-role) of `link`, or a role that inherits from the `link` role, are [included in the accessibility tree](#included-in-the-accessibility-tree), and that have [matching](#matching-text-content) [accessible names](#accessible-name) that contain [text content](#text-content).
+
+### Expectation
+
+When activated, the links in each set of target elements resolve to the [same resource](#same-resource) or to different resources that fulfill an [equivalent purpose](#equivalent-purpose).
```

Comment:
What is meant by "activated"?

Reply:
Suggestions for a better wording for activating/clicking/following?

Reply:
Let's tackle https://github.com/auto-wcag/auto-wcag/pull/220#discussion_r267256695 first and then look at this.

Reply:
https://github.com/auto-wcag/auto-wcag/pull/220#discussion_r267256695 tackled. Any suggestions for this one then?

Reply:
Changed to: "Following the link..."